android/pytorch_android/CMakeLists.txt

```diff
@@ -117,6 +117,8 @@ if(ANDROID_ABI)
   import_static_lib(libeigen_blas)
   import_static_lib(libcpuinfo)
   import_static_lib(libclog)
+  import_static_lib(libfmt)
+  import_static_lib(libkineto)
```

Comment:
Can you please elaborate in PR how kineto is used in Android?

Reply:
same as in other builds? as in - using `torch.profiler` context manager and saving the trace file

Reply:
Hmm, but I don't think Android builds have any Python bindings enabled in them, are they?

Reply:
Probably not, but going forward we should be able to use profiler with c++ APIs.

Reply:
sg, let's keep in mind future C++ api, but for now, as discussed, will split the mobile build support in a separate pr